src/svgcontext.js

```diff
@@ -334,7 +336,7 @@ export class SVGContext {
       height *= -1;
     }
 
-    this.rect(x, y, width - 0.5, height - 0.5, this.attributes);
```

Comment:
Yes, IIRC, the only way you could get 1px lines (without anti-aliasing) at the time was to set the width to 0.5. Not sure if this is still the case.